wazp/pages/04_dashboard.py

```diff
@@ -1,222 +1,35 @@
-    {
-        "ROI": [
-            "ROI 1",
-            "ROI 2",
-            "ROI 3",
-            "ROI 4",
-            "ROI 1",
-            "ROI 2",
-            "ROI 3",
-            "ROI 4",
-        ],
-        "occupancy": [0.4, 0.2, 0.4, 0.5, 0.3, 0.8, 0.7, 0.1],
-        "sex": ["M", "M", "M", "M", "F", "F", "F", "F"],
-    }
-)
-
-fig_barplot = px.bar(
-    df_occupancy,
-    x="ROI",
-    y="occupancy",
-    color="sex",
-    barmode="group",
-    title="Barplot occupancy",
-)
-fig_barplot.update_layout(clickmode="event+select")
-
-
-# Entries/exits matrix
-data = [[10, 25, 30, 50], [20, 60, 80, 30], [30, 60, 15, 20], [80, 50, 20, 39]]
-fig_entries_exits = px.imshow(
-    data,
-    labels=dict(x="departure ROI", y="destination ROI", color="count"),
-    x=["ROI 1", "ROI 2", "ROI 3", "ROI 4"],
-    y=["ROI 1", "ROI 2", "ROI 3", "ROI 4"],
-    title="Entries/exits matrix",
-)
-
-fig_entries_exits.update_layout(
-    clickmode="event+select", height=500, width=500
-)
-fig_entries_exits.update_xaxes(side="top")
-
-
-###############
-# Dashboard layout
-
-
-def plots_first_row_left():
-    """Configure the HTML div contents for the first row left hand side"""
-    trajectories_graph = dcc.Graph(
-        id="graph-trajectories",
-        figure=fig_trajectories,
-        style={
-            "width": "85%",
-            "display": "inline-block",
-        },
-    )
-    radio_button = dcc.RadioItems(
-        ["likelihood", "frame", "input data"],
-        "likelihood",
-        style={
-            "width": "15%",
-            "float": "right",
-            "margin-top": "110px",
-        },
-        labelStyle={
-            "display": "block",
-            "fontSize": ".8rem",
-        },
-    )
-    return html.Div(
-        children=[trajectories_graph, radio_button],
-        style={"width": "49%", "display": "inline-block"},
-    )
-
-
-def plots_first_row_right():
-
-    nbins_button = dcc.Input(
-        id="bin-size",
-        type="number",
-        placeholder="nbins per axis",
-        debounce=True,  # need to press enter,
-        style={"float": "right"},
-    )  # "value" will be in callback
-
-    heatmap_graph = dcc.Graph(
-        id="graph-heatmap",
-        figure=fig_heatmap,
-        style={
-            "width": "85%",
-            "display": "inline-block",
-            "float": "right",
-        },
-    )
-    return html.Div(
-        children=[nbins_button, heatmap_graph],
-        style={"width": "49%", "display": "inline-block"},
-    )
-
-
-def plots_second_row_left():
-    return dcc.Graph(
-        id="graph-barplot",
-        figure=fig_barplot,
-        style={"width": "42%", "display": "inline-block"},
-    )
-
-
-def plots_second_row_right():
-    return dcc.Graph(
-        id="graph-entries-exits",
-        figure=fig_entries_exits,
-        style={
-            "width": "42%",
-            "display": "inline-block",
-            "float": "right",
-        },
-    )
-
```

Comment:
Or is your plan to reimplement them when we do the pose-estimation-import tab?

Reply:
I would hazard the plan was to reimplement them separately, since the first implementation was mock (probably Sofia has a plan for this). 
Personally, I think it's worth keeping a few default plots on the dashboard, at least trajectory and heatmap